ansible/roles/fes-app-config/files/deployment-scripts/frontend_deployment.yml

```diff
@@ -1,4 +1,25 @@
 ---
+  - name: CA Certs
+    hosts: localhost
+    gather_facts: true
+    vars:
+      cert_name: "chca"
+    tasks:
+      - name: Download the cert artifact from S3
+        aws_s3:
+          bucket: "{{ s3_bucket_releases }}"
+          object: "chl-fes/{{ cert_name }}.pem"
+          dest: "/etc/pki/ca-trust/source/anchors/{{ cert_name }}.pem"
+          mode: get
+
+      - name: Update trust
+        shell: /usr/bin/update-ca-trust
+      
+      - name: Force systemd to read all configs
+        systemd:
+          daemon_reload: yes
+      
+
```

Comment:
Theres no need to create a new play here, you can integrate the tasks here into the existing play and put them at the start if required.